pkg/planner/core/plan_cache_utils.go

```diff
@@ -291,7 +291,6 @@ func NewPlanCacheKey(sctx sessionctx.Context, stmt *PlanCacheStmt) (key, binding
 
 	hash := make([]byte, 0, len(stmt.StmtText)*2) // TODO: a Pool for this
 	hash = append(hash, hack.Slice(stmtDB)...)
-	hash = codec.EncodeInt(hash, int64(vars.ConnectionID))
```

Comment:
even the session level plan doesn't encode connection id any more?

Reply:
Yes, each session has its own separated Session Plan Cache, so this connectionID is unnecessary in this case as well.